marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/flow/test-data/jobs/job3.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
For testing, don't you need a job document that has at least two step responses - one for each occurrence of the step in the flow?

Reply:
Right, I think we need those two scenarios above addressed. But depending on the stepNumber - while brittle - seems reasonable for now, since it's what DHF has been doing.

Reply:
job3.json and job4.json addresses scenario 1 and job5.json scenario2.